### ISSUE TITLE
Debian version shown is incorrect

### ISSUE THREAD

**Issue by imrejonk** [Graylog2/graylog2-server#11595]
The Graylog, OS and kernel version shown in the bottom of every web interface page shows an incorrect Debian version.

## Expected Behavior

The text should show Debian version 10.11 on Debian 10.11.

## Current Behavior

The text shows Debian 11.0.12, which is incorrect. Also, if I'm not mistaken, Debian releases don't have a patch level in their version number, i.e. the format is always MAJOR.MINOR.

## Possible Solution

The installed Debian version can be obtained from /etc/debian_version.

## Steps to Reproduce (for bugs)

1. Install Graylog on Debian 10.11.
2. Notice that the web interface shows that Graylog is installed on Debian 11.0.12.

## Your Environment

* Graylog Version: 4.2.1
* Java Version: OpenJDK 11.0.12+7-2~deb10u1
* Elasticsearch Version: 7.10.2
* MongoDB Version: 4.4.10
* Operating System: Debian 10.11.
* Browser version: Firefox 78.15.0esr-1~deb11u1

**Comment by imrejonk** [comment]
Ah! Of course, I didn't notice that. It would be nice if the text could be reworded so that it is more clear that `11.0.12` here refers to the JVM version. Maybe also add the Debian version, could be useful.

**Comment by imrejonk** [comment]
By the way here is the complete text on my Graylog instance:
`Graylog 4.2.1+5442e44 on graylog3.ciphermail.com (Debian 11.0.12 on Linux 4.19.0-18-amd64)`

**Comment by supahgreg** [comment]
In this case `Debian` is actually the `java.vendor` defined in the runtime you're using ((`Tools.getSystemInformation`](https://github.com/Graylog2/graylog2-server/blob/9809b9a1b7c9d4612d9dde8e6eb3dc1e3876d36d/graylog2-server/src/main/java/org/graylog2/plugin/Tools.java#L188-L194)).

**Comment by imrejonk** [comment]
@supahgreg makes sense. I suppose the `java.vendor` is something like `AdoptOpenJDK` for AdoptOpenJDK, which makes it crystal clear what the version string refers to.

Maybe template this to "`java.vendor` JVM version `version`"?
